rare/components/tabs/games/game_info/game_settings.py

```diff
@@ -41,7 +41,7 @@ def find_proton_wrappers():
                 if os.path.exists(proton) and (
                     os.path.exists(compatibilitytool) or os.path.exists(toolmanifest)
                 ):
-                    wrapper = '"' + proton + '" run'
+                    wrapper = f'"{proton}" run'
```

Comment:
I'm not sure I got those quotes right, since there are a lot of them...

Reply:
Seems fine to me.

Reply:
Good, thanks!